test/bdd/features/introduce.feature

```diff
@@ -1,294 +0,0 @@
-
```

Comment:
Why delete?

Reply:
There's no way that it will work. I cannot ignore this file as I did with go files.

Reply:
Can you try following steps and see ?
1. Remove [all](https://github.com/hyperledger/aries-framework-go/blob/master/test/bdd/features/introduce.feature#L8) tag from the feature file.
2. Ignore [introduce](https://github.com/hyperledger/aries-framework-go/blob/master/test/bdd/bddtests_test.go#L41) in the bdd test tags.ie, `tags := "all,~introduce"`

Reply:
Thanks, it works! I was enough to remove tag `all`